build.sbt

```
@@ -460,7 +460,7 @@ lazy val compilerOptionsExporter = Project("compilerOptionsExporter", file(".")
   .settings(disablePublishing)
   .settings(
     libraryDependencies ++= {
-      val jacksonVersion = "2.19.2"
+      val jacksonVersion = "2.20.0"
       Seq(
         "com.fasterxml.jackson.core" % "jackson-core" % jacksonVersion,
         "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonVersion,
```

jackson-annotations changed its versioning scheme change, see https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.20#compatibility-versioning.

```suggestion
        "com.fasterxml.jackson.core" % "jackson-annotations" % "2.20",
```